picard/ui/coverartbox.py

```diff
@@ -414,10 +425,14 @@ def load_remote_image(self, url, mime, data):
 
         if config.setting["load_image_behavior"] == 'replace':
             set_image = set_image_replace
+            debug_info = "Replacing with dropped %r in %r"
         else:
             set_image = set_image_append
+            debug_info = "Appending dropped %r to %r"
 
+        update = True
         if isinstance(self.item, Album):
+            log.debug(debug_info, coverartimage, self.item)
```

Comment:
This debug line gets repeated for every `isinstance` check, can we move it before the `if`?

Reply:
Yup, i converted a (useless) warning to debug, and simplfied. See 
7dd6cf6